packages/server/lambdas/alfresco/fileRequestBuilder/index.ts

```diff
@@ -18,6 +18,14 @@ export const updateFileRequestBuilder = (event: ALBEvent, headers: HeadersInit,
       return alfrescoRequestBuilder.updateRequestBuilder(event, headers);
   }
 };
+export const updateFileMetadataRequestBuilder = (event: ALBEvent, headers: HeadersInit, store?: FileStore) => {
+  switch (store) {
+    case FileStore.ALFRESCO:
+    default:
```

Comment:
Wonder why these lines are not listed under `case FileStore.ALFRESCO:` but `default:`?

Reply:
Actually, it's both. With no break or return in a case, the switch won't stop going over the list. So this is kinda like `if (case === FileStore.ALFRESCO || case === default) { /* DO STUFF */}`